bluesky_adaptive/agents/base.py

```diff
@@ -464,6 +473,10 @@ def _add_to_queue(self, next_points, uid):
         next_points : Iterable
             New points to measure
         uid : str
+        re_manager : Optional[bluesky_queueserver_api.api_threads.API_Threads_Mixin]
+            Defaults to self.re_manager
+        position : Optional[Union[int, Literal[&quot;front&quot;, &quot;back&quot;]]]
```

Comment:
I would just use single quotes `'` here.

Reply:
Fixed in https://github.com/bluesky/bluesky-adaptive/pull/12/commits/481d4e98270f54c8155b85211eb325e7d4b0e3b8.